core/server.go

```diff
@@ -144,9 +130,8 @@ func WithGrpcManager(manager GrpcManager) ServerOption {
 }
 
 //nolint:gocognit // Long method names are okay because only generated code will call this, not humans.
-func Server(ctx context.Context, name string, hl Manager, grpcHl GrpcManager, logger *logrus.Logger, promRegistry *prometheus.Registry) error {
+func Server(ctx context.Context, name string, hl Manager, grpcHl GrpcManager, promRegistry *prometheus.Registry) error {
```

Comment:
This function needs to remain backwards compatible, meaning `ServerParams` needs to continue to support `WithLogrusLogger`.